src/views/common/Animation/AnimationStateReducer.tsx

```diff
@@ -187,6 +194,13 @@ const doTick = <T, >(s: AnimationState<T>, a: AnimationStateTickAction): Animati
         console.warn('Attempt to animate, but no dispatch function set. No-op.')
         return s
     }
+    if (!s.isPlaying) {
+        // Shouldn't happen, but these sometimes slip through on rapid interactions.
+        console.log(`Caught unexpected tick while theoretically paused.`)
+        s.pendingFrameCode && window.cancelAnimationFrame(s.pendingFrameCode)
+        s.pendingFrameCode = undefined
+        return {...s}
+    }
```

Comment:
This also manually corrects for the race condition by ensuring the animation callback chain will stop itself if the state toggles to not-playing. (Previously this was handled only by the code that toggled playback state.) It's still possible to trip this up by clicking rapidly on the play/pause button, so we might want to do some more formal debouncing, but this seems to work pretty well now.